src/03_modules.py

```diff
@@ -10,22 +10,30 @@
 
 # Print out the command line arguments in sys.argv, one per line:
 # YOUR CODE HERE
+print(sys.argv[0])
 
 # Print out the OS platform you're using:
 # YOUR CODE HERE
+import platform
+
+print(platform.platform())
```

Comment:
you could do this w/o importing platform by printing sys.platform. But using platform works also.